src/procedures/SetDividendsWallet.ts

```diff
@@ -8,9 +8,18 @@ import {
 } from '../types';
 import { PolymathError } from '../PolymathError';
 
+/**
+ * Procedure that updates the wallet address witch receive reclaimed dividends and tax
```

Comment:
Procedure that modifies the wallet address where reclaimed dividends and withheld taxes are kept